<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>导入目录</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTextBrowser" name="textBrowser_file">
     <property name="placeholderText">
      <string>文件列表</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>一键生成</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>自定义格式串</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTextBrowser" name="textBrowserShow">
     <property name="placeholderText">
      <string>文件预览框</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>操作说明：
1、选择包含*.proto后缀的文件目录      
2、在【自定义格式串】中,录入*符号，作为占位符      
3、点击【一键生成】
例如：
录入msg.ProcessorProto.SetRouter(protoMsg.*{}, game.ChanRPC)
则生成msg.ProcessorProto.SetRouter(protoMsg.GameBaccaratBetResult{}, game.ChanRPC)

</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
